support-config/src/main/resources/touchpoint.PROD.conf

```diff
@@ -50,6 +50,10 @@ touchpoint.backend.environments {
         monthly.productRatePlanChargeId = "2c92a0fb4edd70c9014edeaa50342192"
         annual.productRatePlanChargeId = "2c92a0fb4edd70c9014edeaa5001218c"
       }
+      supporterplus {
+        monthly.productRatePlanChargeId = "8a12865b8219d9b401822106194e64e3"
+        annual.productRatePlanChargeId = "8a12865b8219d9b40182210618c664c1"
+      }
```

Comment:
Not one for this PR, but we're just losing compile safety by putting them in a config file rather than scala code, as we would never change these without recompiling and redeploying the whole thing anyway.

Reply:
Yes I agree, this is how it has been done for  the other products though and it would be quite a big refactor to switch them all over. Worth doing though at some point